cdap-app-templates/cdap-etl/cdap-etl-archetypes/cdap-etl-batch-sink-archetype/src/main/resources/archetype-resources/pom.xml

```diff
@@ -29,6 +29,12 @@
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <cdap.version>3.3.1-SNAPSHOT</cdap.version>
     <hadoop.version>2.3.0</hadoop.version>
+    <!-- properties for script build step that creates the config files for the artifacts -->
+    <widgets.dir>widgets</widgets.dir>
+    <docs.dir>docs</docs.dir>
+    <etl.versionRange>[3.3.0-SNAPSHOT,3.4.0-SNAPSHOT)</etl.versionRange>
```

Comment:
thats true.  Maybe since this is an archetype I will just open the upper range to something high